src/rpc/multipass.proto

```diff
@@ -217,16 +217,6 @@ message TargetPathInfo {
     string target_path = 2;
 }
 
-message MountError {
```

Comment:
are we OK with backward incompatible changes of the protocol? Do we need to consider client-server version mismatches? Perhaps we should have a deprecation period?

Reply:
I'm of the opinion that until we draw the 1.0 line in the sand, the rpc protocol stability is off the table.  Anyone who has created a client using the rpc itself will just have to fix their client.

@Saviq, @gerboland, any opinions?

Reply:
Yeah, we don't consider the protocol stable until we make it official. Also client/server split is not really possible yet, so +1 here.